src/components/MovieCard/MovieCard.tsx

```diff
@@ -6,39 +6,61 @@ type Props = {
   movie: Movie,
 };
 
-export const MovieCard: React.FC<Props> = ({ movie }) => (
-  <div className="card" data-cy="movieCard">
-    <div className="card-image">
-      <figure className="image is-4by3">
-        <img
-          data-cy="moviePoster"
-          src={movie.imgUrl}
-          alt="Film logo"
-        />
-      </figure>
-    </div>
-    <div className="card-content">
-      <div className="media">
-        <div className="media-left">
-          <figure className="image is-48x48">
+export const MovieCard: React.FC<Props> = ({ movie }) => {
+  const {
+    imgUrl,
+    title,
+    description,
+    imdbUrl,
+  } = movie;
+
+  const isTitle = (imgUrl !== 'N/A');
```

Comment:
It should has `hasImage` name, I think. It this about the image, not the title, am I right?

Reply:
Ooops =)